### ISSUE TITLE
Campaign Landing Page request: Legacy sign in credential migration

### ISSUE THREAD

**Issue by CarlDickerson** [department-of-veterans-affairs/va.gov-cms#13603]
## Description
As part of VA.gov's Campaign Landing Page, the CEDAR Identity Login.gov Adoption Team wants to launch a CLP to help provide  updates on coming changes to how users sign in to VA. As VA transitions away from the use of DS Logon and My HealtheVet as valid sign in credentials for accessing VA websites and applications, this CLP would serve as resources to provide eduction around VA's new preferred credential--Login.gov, support on setting up a new account, and other information they may need to know surrounding the new secure sign in experience. 

## Intake
- [x] What triggered this runbook? (Help desk ticket, Product team, Slack thread, etc)
Trigger: CEDAR Identity Login.gov adoption product team

- [x] Link to associated JIRA help desk ticket (if applicable)
Help desk ticket: https://va-gov.atlassian.net/browse/VAHELP-5758

- [x] Vanity URL requested:
https://www.va.gov/secure_sign_in

- [x] Name of submitter (if applicable)
Submitter: Carl Dickerson

## Acceptance criteria
  
- Request comes in, via helpdesk request or direct to PO, or potentially in a Github issue.
- [x] If not in a helpdesk request, redirect to Helpdesk or create a Helpdesk issue using email support@va-gov.atlassian.net or URL https://prod.cms.va.gov/help
Helpdesk will use #sitewide-public-websites channel to request approvals
- [x] In helpdesk ticket, @ mention Dave Conlon to review/approve the creation / existence / description of the CLP. Example text: 
> Hi Dave, Helpdesk has received a request for a new Campaign Landing Page. Can you confirm that this CLP is approved?

#### Helpdesk steps
- [x] Helpdesk or any Drupal Admin can create the CLP node. Clone the [CLP template node](https://prod.cms.va.gov/node/16512) 
https://prod.cms.va.gov/initiatives/secure-sign-in
- [x] If a new CMS editor user is needed, Helpdesk or any Drupal admin can create them and assign proper permissions according to [CLP CMS account administration policy](https://prod.cms.va.gov/help/cms-account-admin-policies/clp-cms-account-administration-policy).
- [x] Send the editor a link to the KB article: [self-guided Campaign Landing Page training](https://prod.cms.va.gov/help/campaign-landing-pages/how-to-manage-campaign-landing-pages)
- [x] Notify #sitewide-public-websites team "New CLP requested and approved" with link to this Github issue for next steps.

#### Public Websites steps
- [x] Create a [Vanity URL redirect issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&template=redirect-request.md&title=Redirect+Request), IA _must_ review & approve URL requests.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/58142

- [x] @mention Mikki Northuis in IA (#sitewide-content-ia) with link to Github issue, to provide Vanity URL request feedback. IA _must_ review & approve URL requests.

- [ ] When approved by IA, execute the steps of the URL change request ticket from step 2 above.

- [ ] When redirect is verified in Production, close Redirect ticket as well as this ticket.

**Comment by jilladams** [comment]
Hi @CarlDickerson ! Thanks for the ticket. Could you follow the step in ticket to request the CLP creation from the Helpdesk team? They'll create the page, offer training if needed, get any users created in the CMS that might be necessary, and send you back to us after.   (FYI @stefaniefgray )

**Comment by CarlDickerson** [comment]
> Hi @CarlDickerson ! Thanks for the ticket. Could you follow the step in ticket to request the CLP creation from the Helpdesk team? They'll create the page, offer training if needed, get any users created in the CMS that might be necessary, and send you back to us after. (FYI @stefaniefgray )
> 
> > redirect to Helpdesk or create a Helpdesk issue using email [support@va-gov.atlassian.net](mailto:support@va-gov.atlassian.net) or URL https://prod.cms.va.gov/help
> > Helpdesk will use #sitewide-public-websites channel to request approvals

@jilladams Helpdesk ticket was submitted last night, same time as Github request. VAHELP-5758 Campaign Landing Page request: Legacy sign in credential migration

For visibility @elizabethkochBT

**Comment by stefaniefgray** [comment]
Wrote to ask Dave C. if the request is approved, waiting on response

**Comment by CarlDickerson** [comment]
@stefaniefgray I see movement on this ticket, thank you. Once Dave C approves, is there a usual time frame that we can expect before we can start building out the CLP?

**Comment by jilladams** [comment]
Note: Dave's on PTO til Tues 5/9.

**Comment by stefaniefgray** [comment]
CLP created and KB article + draft template sent to editors earlier today in linked Jira ticket

**Comment by mnorthuis** [comment]
Hi all,   because this is part of the broader comms of the identity work, please work with @DanielleThierryUSDSVA and Sitewide on this one.  I'm going to put the vanity URL request on hold until this campaign content is more framed up and we can choose a vanity URL that reflects that messaging and the overall comms.  We should have plenty of time to do that.

**Comment by CarlDickerson** [comment]
> Hi all, because this is part of the broader comms of the identity work, please work with @DanielleThierryUSDSVA and Sitewide on this one. I'm going to put the vanity URL request on hold until this campaign content is more framed up and we can choose a vanity URL that reflects that messaging and the overall comms. We should have plenty of time to do that.

Thanks @mnorthuis Mikki, will do!

**Comment by jilladams** [comment]
Removed from PW sprints til we get the green light on redirect.

**Comment by jilladams** [comment]
@mnorthuis @DanielleThierryUSDSVA @CarlDickerson just scrubbing our backlog: Any updates here, or is this Legacy sign in creds migration CLP still blocked on greater identity work til we do the redirect?

**Comment by CarlDickerson** [comment]
> FYSA: @DanielleThierryUSDSVA we got a new ping on this CLP today via #16271 (duplicate ticket). I'm working on confirming with @TroyA6 / Helpdesk what they've heard from Carl Dickerson in JIRA, but meantime: let us know if any updates from your perspective, here. Thanks!

@jilladams I put in the request for a new CLP this morning. I wasn't sure if I should fill out the JIRA ticket yet until I got a confirmation, but I will go ahead and do that. I tried to annotate in the request that it was not duplicate per see. As our comms plan has changed, we're requesting a new CLP with a new URL. As we've been aligned, Danielle should be able to support/elaborate on this request.

**Comment by jilladams** [comment]
Ok. If someone can please advise re: the previous draft CLP as well, whether it will be "remodeled" to become the new CLP or should be archived, we can go from there. Thanks!

**Comment by TroyA6** [comment]
@jilladams  - I added this note to the other GH ticket as well, but this is all we received from Carl regarding this request:
Components

[Campaign Landing Page](https://va-gov.atlassian.net/issues/?jql=project%20%3D%20%22VAHELP%22%20AND%20component%20%3D%20%22Campaign%20Landing%20Page%22)
Where on the site is the issue occurring? (Please paste a direct link - thank you!)

https://www.va.gov/initiatives/sign-in-securely-with-logingov/
Description

Hi,

I am a member of VA OCTO [Login.gov](http://login.gov/) adoption team. We would like to request a new CLP to support our ongoing work to migrate Veterans to more secure sign-in credentials. This work is supported by Danielle Thierry and Dave Conlon (we have approvals on both ends). 

Can you please CC replies to [Lauren.Morris3@va.gov](mailto:Lauren.Morris@va.gov) and [Sophia.Philip@VA.gov](mailto:Sophia.Philip@VA.gov).

**Comment by TroyA6** [comment]
@jilladams - Dave C commented that further approval is not needed, as this is coming from CAIA - documented in [Slack ](https://dsva.slack.com/archives/C52CL1PKQ/p1701459723628929)

**Comment by jilladams** [comment]
I'm wrong, just easily confused: 
* CLP created here for #13603 = https://prod.cms.va.gov/initiatives/secure-sign-in, published
* CLP created for #16271 = https://prod.cms.va.gov/initiatives/migrating-veterans-accounts-to-more-secure-sign-in-credentials

Reopening 16271.

**Comment by jilladams** [comment]
Sorry, more questions: 
* Current published CLP = https://prod.cms.va.gov/initiatives/secure-sign-in
* New CLP in draft = https://prod.cms.va.gov/initiatives/migrating-veterans-accounts-to-more-secure-sign-in-credentials

My questions:
1. @DanielleThierryUSDSVA in your note above, do I understand that you want to redirect the published CLP to the new CLP, when the new one gets published? Dave has previously suggested that CLPs should be time-limited, so I'm wondering if that factors into the redirect idea at all. 
2. @CarlDickerson you filed #16275 on Friday as well, and I'm trying to understand how that connects or doesn't connect to these 2 CLP tickets. I added a note on that ticket for clarity as well -- feel free to answer there, just flagging here for everyone's awareness.

**Comment by CarlDickerson** [comment]
@jilladams I answered in #16275 but I will also answer here for continuity. #16275 is a duplicate request of #16271. Not realizing that my HelpDesk request would trigger a Github issue, I also went and created ticket for this work myself. So I think that #16275 can be closed since the work is being tracked elsewhere.

**Comment by jilladams** [comment]
Thanks very much. Now that we have that cleared up, I'll use #16271 for any remaining questions. The CLP that was created via this ticket (13603) has been published (9/25/23), so I'm closing this ticket as well.
